src/rules/padding.ts

```diff
@@ -0,0 +1,384 @@
+/**
+ * @fileoverview Rule to require or disallow newlines between jest functions,
+ *   based on eslint/padding-line-between-statements by Toru Nagashima
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const astUtils = require('eslint/lib/util/ast-utils');
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Check if we're in a test file
+ *
+ * @param filename {string}
+ */
+function isTestFile(filename) {
```

Comment:
Can we add typing for this

Reply:
Just one type annotation in particular was giving me trouble so I left it as an implicit `any`. The `verify` function within `create` was the troublemaker. I expected it to take an `ESTree.Node`, but... no.

Reply:
I'll mark this comment resolved, but I'm still working a bit on some additional clean up opportunities in `padding.ts`. I'm going to take one more try on that type, too.